tests/components/habitica/conftest.py

```diff
@@ -21,6 +24,28 @@ def disable_plumbum():
         yield
 
 
+def assert_mock_called_with(
```

Comment:
Test helpers that have asserts embedded like this can make tests a little difficult to follow since you have to know what the helper is asserting on.  A similar pattern can be something like:

```
    mock_call = get_mock_call(
        mock_habitica, "post", f"{DEFAULT_URL}/api/v3/tasks/{uid}/score/up"
    )
    assert mock_call
```

Reply:
That sounds like a very good idea, thanks 👍